test/runtests.jl

```diff
@@ -163,6 +163,54 @@ mkpath(TEST_EXAMPLES_DIR)
       @test sum(data) ≈ 192.1204941112099
     end
   end
+
+  # ----------------------------
+  # Test for validation of structured grid read feature
+  # ----------------------------
```

Comment:
```suggestion
  # Test for validation of uniform grid ("image data") read feature
```
Just to be clear that we do not have support for general structured meshes yet

Reply:
good clarification